notebooks/metatranscriptomics_pipeline_iSeqnorm.ipynb

```diff
@@ -0,0 +1,1477 @@
+    "\n",
+    "if not os.path.isfile(plate_map_fp):\n",
+    "    print(\"Problem! %s is not a path to a valid file\" % file)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Read in the plate map**. It should look something like this:\n",
+    "\n",
+    "```\n",
+    "Sample\tRow\tCol\tBlank\n",
+    "GLY_01_012\tA\t1\tFalse\n",
+    "GLY_14_034\tB\t1\tFalse\n",
+    "GLY_11_007\tC\t1\tFalse\n",
+    "GLY_28_018\tD\t1\tFalse\n",
+    "GLY_25_003\tE\t1\tFalse\n",
+    "GLY_06_106\tF\t1\tFalse\n",
+    "GLY_07_011\tG\t1\tFalse\n",
+    "GLY_18_043\tH\t1\tFalse\n",
+    "GLY_28_004\tI\t1\tFalse\n",
+    "```\n",
+    "**Make sure there a no duplicate IDs**. If each sample doesn't have a different name, an **error** will be thrown and you won't be able to generate a sample sheet."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "##need to specify the directory for qiita.oauth.cfg\n",
+    "plate_df = read_plate_map_csv(plate_map_fp)\n",
+    "plate_df.head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Step 2: read in DNA/RNA concentrations and add to plate map\n",
+    "\n",
+    "**Enter the correct path to the Pico DNA concentration output**. This should be a txt-formatted file produced by the MiniPico assay on the condensed, 384-well plate. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sample_dna_concs_fp = './test_data/Quant/MiniPico/2024_NPH_Plates_3-5_gDNA_quant.txt'\n",
+    "\n",
+    "if not os.path.isfile(sample_dna_concs_fp):\n",
+    "    print(\"Problem! %s is not a path to a valid file\" % file)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Read in the DNA concentration output file**. It should be a .txt file and should look something like this:\n",
+    "    \n",
+    "```\n",
+    "##BLOCKS= 1\r\n",
+    "Group: Unknowns\r\n",
+    "Sample\tWells\tRFU_Values\tConcentration\tMean_Conc\tSD\tCV\tDilution\tAdjConc\t\r\n",
+    "01\tA1\t16352.000\t0.434\t0.434\t0.000\t0.0\t\t\t\r\n",
+    "02\tC1\t26061758.000\t37.588\t37.588\t0.000\t0.0\t\t\t\r\n",
+    "03\tE1\t1483519.000\t2.527\t2.527\t0.000\t0.0\t\t\t\r\n",
+    "04\tG1\t3015184.000\t4.712\t4.712\t0.000\t0.0\t\t\t\r\n",
+    "05\tI1\t16574.000\t0.434\t0.434\t0.000\t0.0\t\t\t\r\n",
+    "06\tK1\t14988.000\t0.432\t0.432\t0.000\t0.0\t\t\t\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sample_dna_concs = read_pico_csv(sample_dna_concs_fp, plate_reader='SpectraMax_i3x',conc_col_name='Sample DNA Concentration')\n",
+    "\n",
+    "plate_df = pd.merge(plate_df, sample_dna_concs, on='Well')\n",
+    "\n",
+    "well_col = 'Well'\n",
```

Comment:
Why is this defined here but not used in this block?

Reply:
I can clean this up, thanks for pointing this out.